Cyberduck/Connection.md

```diff
@@ -148,6 +148,20 @@ Integrated Windows Authentication (IWA) for proxy authentication of HTTP connect
 
 # Preferences
 
+## General → Connection → Use Keychain
+
+Search for and save password for connections.
+
+## Profiles
+
+```{note}
+Prelimitary documentation for Cyberduck 8
+```
+
+Select connection protocols to be installed in addition to the default protocols. The connection profile will be installed after enabling the corresponding checkbox. To disable a connection profile simply uncheck the checkbox. The profile will be disabled after closing the application.
+
+**Note:** You can't disable the default protocols. If you disable the connection profile used in a bookmark the bookmark won't work after restarting the application.
```

Comment:
Set the sentence into the admonition note instead of using "**Note:**" as prefix.